changelogs/unreleased/gh-7362-net-box-async-ping.md

```diff
@@ -0,0 +1,7 @@
+## feature/core
+
+* Introduced `is_async` option for `conn:ping()`. If the corresponding
+  option is passed to `ping()` future object will be returned. As soon
+  as future is ready its result will be either true or nil + error,
+  which is similar to the behavior of the future object in all net.box's
+  methods.
```

Comment:
Nit: Not necessary to go into details in a changelog - there's a documentation for that. I think that the first sentence alone would be enough.

Reply:
Done